frontend/src/components/messages/Messages.styled.ts

```diff
@@ -14,8 +16,8 @@ const MessagesWrapper = styled.div`
     }
 
     &::-webkit-scrollbar-thumb {
-        background: #313131;
         border-radius: 3px;
+        background: #050050;
```

Comment:
IMO each color should be a const in theme file

Reply:
ok, done

Reply:
done